archinstall/lib/installer.py

```diff
@@ -1054,17 +1055,15 @@ def enable_sudo(self, entity: str, group :bool = False) -> bool:
 
 		return True
 
-	def user_create(self, user :str, password :Optional[str] = None, groups :Optional[str] = None, sudo :bool = False) -> None:
-		if groups is None:
-			groups = []
-
+	def user_create(self, users: List[User]) -> None:
```

Comment:
The function should stay as it were, in the sense that it was a function to perform one action.
It made it's purpose a bit more clear. And I propose we create a secondary function that deals with the loops and stuff, which would be called `load_users_layout()` or something similar to `load_disk_layout()`.
Thoughts?

Reply:
Yes that's a great point! I'll update that one :)